### PR TITLE
refactor: delete unused commands in tablet client

### LINKED ISSUE #811: Delete unused commands in tablet client
The following commands should be deleted in [StartClient](https://github.com/4paradigm/OpenMLDB/blob/main/src/cmd/openmldb.cc#L4378)
* `create` 
* `put` 
* `sput` 
* `get` 
* `scan` 
* `benchmark`  
* `benget` 
* `benput` 
* `benscan`  

**Note**: Relevant help parameter and usage should be delete in  [HandleClientHelp](https://github.com/4paradigm/OpenMLDB/blob/main/src/cmd/openmldb.cc#L3266)

### PR DESCRIPTION
**Pull request by nautaa**
* **Please check if the PR fulfills these requirements**

- The commit message follows our guidelines
- Tests for the changes have been added (for bug fixes / features)
- Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
refactor: delete unused commands in tablet client

* **What is the current behavior?** (You can also link to an open issue here)
close #811

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

**Review by superzhaozheng — APPROVED** [review]
LGTM

File: src/cmd/openmldb.cc [hunk 1]
```diff
@@ -4435,14 +4393,6 @@ void StartClient() {
...
         } else if (parts[0] == "getfollower") {
             HandleClientGetFollower(parts, &client);
-        } else if (parts[0] == "benput") {
-            HandleClientBenPut(parts, &client);
-        } else if (parts[0] == "benscan") {
-            HandleClientBenScan(parts, &client);
-        } else if (parts[0] == "benget") {
-            HandleClientBenGet(parts, &client);
-        } else if (parts[0] == "benchmark") {
-            HandleClientBenchmark(&client);
```
Comment (dl239) [inline src/cmd/openmldb.cc:4445]: Hi, @nautaa thanks for your pr.  
The implementation of these functions also needs to be deleted together.

Comment (nautaa) [reply src/cmd/openmldb.cc:4445]: @dl239 Done.

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Review by dl239 — APPROVED** [review]

**Comment by lumianph** [comment]
[community pr/issue tracking](https://trello.com/c/S4D8P57j/17-community-pr-issue-tracking)

**Pull request merged**